packages/web3-providers-http/src/index.js

```diff
@@ -27,10 +27,15 @@ var errors = require('web3-core-helpers').errors;
 var http = require('http');
 var https = require('https');
 
+// check if node isn't being run
+if (!typeof window === undefined) {
 // Apply missing polyfill for IE
 require('cross-fetch/polyfill');
 require('es6-promise').polyfill();
 require('abortcontroller-polyfill/dist/polyfill-patch-fetch');
+} else {
+    const fetch = require('cross-fetch');
+}
```

Comment:
Does this work the way you want it to? `fetch` is later accessed from global scope.

Reply:
Your right, fetch was still being undefined. removed const, thanks